libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -394,8 +394,6 @@ public void onServiceConnected(ComponentName className, IBinder service) {
 
       @Override
       public void onServiceDisconnected(ComponentName className) {
-        telemetryService = null;
```

Comment:
this is still required IMHO for the cases when connection with the service is unexpectedly interrupted.